lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -396,7 +397,11 @@ func (handler *ContainersHandlersImpl) GetContainerLogsHandler(params containers
 		tail = int(*params.Taillines)
 	}
 
-	reader, err := container.LogReader(context.Background(), tail, follow)
+	if *params.Since > 0 {
+		since = *params.Since
+	}
+
```

Comment:
do we care about this check? if params.Since is 0 than that's OK to as since's default is 0. I think you can remove the if check altogether